### PR TITLE
Update jet support

### LINKED ISSUE #770: Update Jet Support
Jet has a new location for hpc-stack builds that is managed by EPIC.  In addition, the unit/regression tests should be enabled on that platform.

### PR DESCRIPTION
**Pull request by DavidHuber-NOAA**
## DESCRIPTION OF CHANGES: 
This updates Jet support by pointing the module file to the new EPIC-managed HPC-stack build location.  An mpiexec.jet was also created to enable testing on the platform.  Note that connections to the EMC FTP site are not allowed yet, though they have been requested.

## TESTS CONDUCTED: 
All unit tests were run on Jet and passed.  Note that the data had to be downloaded on Hera then pushed to Jet and staged before running the tests.

## DEPENDENCIES:
None

## ISSUE: 
- Fixes #770 
- NOAA-EMC/global-workflow#357

### CODE REVIEW

File: cmake/mpiexec.jet [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```
Comment (GeorgeGayno-NOAA) [inline cmake/mpiexec.jet:1]: If you can't download the test data to Jet, how did you test this file?

Comment (DavidHuber-NOAA) [reply cmake/mpiexec.jet:1]: I transferred the data from Hera and staged it into the build/tests/*/data directories.

**Comment by GeorgeGayno-NOAA** [comment]
Ran the regression tests on Jet using the branch at 4a9e1f1. All tests passed.

File: cmake/mpiexec.jet (hunk 1 above)
Comment (GeorgeGayno-NOAA) [reply cmake/mpiexec.jet:1]: Ok. I commented out the ftp of the test data. The file did successfully kick off all the jobs (although some of them failed as expected.) When do you expect ftp access from Jet?

Comment (DavidHuber-NOAA) [reply cmake/mpiexec.jet:1]: I don't know how long these things take, but I submitted a request that was acknowledged on 2/3.  Would you like to be added to the ticket?

Comment (GeorgeGayno-NOAA) [reply cmake/mpiexec.jet:1]: That is not necessary. Just curious. Once access is granted, we might want to update the build_all.sh script.

**Review by GeorgeGayno-NOAA — APPROVED** [review]
Changes look good. Will merge.

**Pull request merged**